src/ray/object_manager/object_manager.h

```diff
@@ -32,6 +32,7 @@
 #include "ray/common/id.h"
 #include "ray/common/ray_config.h"
 #include "ray/common/status.h"
+#include "ray/gcs/gcs_client/service_based_gcs_client.h"
```

Comment:
Why we need this line?

Reply:
oh, we don't need `gcs client` in `object_manager.h`, will delete,